packages/eslint-plugin-office-addins/src/rules/no-navigational-load.ts

```diff
@@ -43,6 +43,9 @@ export = {
         }
       }
 
+      if (lastProperty === "*") {
+        return true;
+      }
```

Comment:
It should be after the loop because I want to catch cases like:
`range.load("font/size/*")` so we still need to validate that `font` and `size` are navigational properties.

Reply:
Have you checked that '*' at the end of the path like that is valid (by trying the code out)?

Reply:
I just tried and it is valid.

Reply:
Good to know.